warehouse/upload.go

```diff
@@ -1098,7 +1096,14 @@ func (job *UploadJobT) loadTable(tName string) (alteredSchema bool, err error) {
 		var errTotalCount error
 		totalBeforeLoad, errTotalCount = job.getTotalCount(tName)
 		if errTotalCount != nil {
-			pkgLogger.Errorf(`Error getting total count in table:%s before load: %v`, tName, errTotalCount)
+			pkgLogger.Warnw("total count in table before loading",
+				warehouseutils.SourceID, job.upload.SourceID,
+				warehouseutils.DestinationID, job.upload.DestinationID,
+				warehouseutils.DestinationType, job.upload.DestinationType,
+				warehouseutils.WorkspaceID, job.upload.WorkspaceID,
+				warehouseutils.ERROR, errTotalCount,
+				warehouseutils.TABLE_NAME, tName,
+			)
```

Comment:
[minor] The upper case usage is not idiomatic to go.

It might be a good idea to introduce a separate package for standard log fields, to separate them from the application-specific constants.

```suggestion
				logfield.Error, errTotalCount,
				logfield.TableName, tName,
			)
```

Reply:
Good idea.